src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -705,7 +719,10 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
 
             //PROTOTYPE: should be a wellknown member?
             var ctor = type.UnderlyingTupleType?.InstanceConstructors.FirstOrDefault();
```

Comment:
The helper will take care of reporting appropriate diagnostics. So, the throw below will be unnecessary.